src/wmtk/attribute/TupleAccessor.hpp

```diff
@@ -19,6 +25,9 @@ class TupleAccessor : protected CachingAccessor<T>
     friend class wmtk::Mesh;
     friend class wmtk::TetMesh;
     friend class wmtk::TriMesh;
+    friend class wmtk::EdgeMesh;
+    friend class wmtk::PointMesh;
+    friend class wmtk::TriMeshOperationExecutor;
```

Comment:
It's a bit odd that `TriMeshOperationExecutor` needs to be a friend but `TetMeshOperationExecutor` does not.

Reply:
yeah, i hadn't thought about that, but for whatever reason this was required